fake-ppud-automation-api/__files/get-offender-X098095.json

```diff
@@ -32,19 +32,6 @@
           "indexOffence": "Blasphemy against coca cola",
           "dateOfIndexOffence": "2008-06-12"
         },
-        "releases": [
-          {
-            "category": "Not Applicable",
-            "dateOfRelease": "2013-02-02",
```

Comment:
Should we not be keeping this as the new `releaseDate` field? Or do you deliberately want it not to have one?